databases/timescaledb/Portfile

```diff
@@ -101,11 +101,16 @@ variant timescale_license description {Enable Timescale License code, license wi
 }
 
 if {${name} eq ${subport}} {
-    PortGroup       obsolete 1.0
-    revision        1
-
+    # make it a stub port like py-foo
```

Comment:
The stub portgroup will empty `distfiles`, which will lead to void `port livecheck` or `port bump`, therefore (I guess) the python packages (via python portgroup) are using its own stub implement instead of importing stub portgroup. I think we should follow this too.